test/mix/tasks/phx.gen.auth_test.exs

```diff
@@ -167,7 +167,8 @@ defmodule Mix.Tasks.Phx.Gen.AuthTest do
             delete "/users/log_out", UserSessionController, :delete
```

Comment:
Users backporting this change to their apps can ignore the changes to this file.